### DISCUSSION TITLE
Encrypted Partition

### DISCUSSION THREAD

**Discussion by lawmanuk** [helloSystem/hello#555] [General]
Hi

I want to test this out by installing on a partition encrypted.

At present there seems to only whole disk installation and no encryption.

Is there a way to resolve this currently, or feature to consider for future?

thanks

**Comment by probonopd** [comment]
Hi, our installer currently cannot do this.
But contributions in this are definitely would be very welcome.

The relevant code is in
https://github.com/helloSystem/ISO/blob/e9294ba7d6e437a0402efb374a5943ea3c90e0db/overlays/uzip/furybsd-live-settings/files/usr/local/bin/furybsd-install#L104
